ext/RMagick/rmimage.c

```diff
@@ -15525,6 +15525,23 @@ static void call_trace_proc(Image *image, const char *which)
 }
 
 
+static VALUE
+rm_trace_creation_body(VALUE img)
```

Comment:
Just to confirm my understanding. `rb_rescue` only accepts a `VALUE` type, right? Otherwise, we could pass an `Image *image` here and avoid casting below.

Reply:
The some Ruby C API (like`rb_rescue`, `rb_exec_XXXX`) will pass the argument directly to our function (`rm_trace_creation_body()`, `rm_trace_creation_handle_exception()`).
So, we can use any type argument in here with carefully. 

We can often see the same technique in Ruby core. https://github.com/ruby/ruby/blob/b1daba8227ef420247e0133ecf9bb5b647b891e0/io.c#L11402

Reply:
To just remove compiler warning message, I used `VALUE` type in here.

Reply:
I see. Thanks for the explanation!